test/test_worker_gem_independence.rb

```diff
@@ -142,4 +145,10 @@ def with_unbundled_env
       Bundler.with_unbundled_env { yield }
     end
   end
+
```

Comment:
@dentarg 

Added testcases in `TestWorkerGemIndependence`. not sure if it should be in separate or new class.

also used grep here as I did not find any alternative to get the title of Process running. we are using `setproctitle` to set the title but there is not equivalent for this to get.